data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,173 @@
+	on offer
+		conversation
+			`Eeeya takes you to the bunker where the Cloudfire military base commander resides. Unfortunately, the soldiers won't let you in. After about an hour of Eeeya's panicking, they agree to let her speak with the commander, as long as you wait outside.`
+			`	After a few minutes, she comes out despondent and you lead her back to the ship.`
+			`	"They said she doesn't work for them. I think they know where she is, but they won't tell me anything.`
+			`	"There's one more place we can try, but I don't think you're going to like it. She may have joined the Unfettered Hai."`
+			label questions
+			choice
+				`	"Why didn't you mention this earlier?"`
+					goto mention
+				`	"Why would a doctor join a band of warmongers?"`
+					goto warmonger
+				`	"Let's go find her!"`
+					goto accept
+				`	"Unfettered space is too dangerous for me. You will have to find another captain."`
+					decline
+			label mention
+			`	"I was afraid it would scare you away. Humans are terrified of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely."`
+				goto questions
+			label warmonger
+			`	"Anoo was always sympathetic to the cause of the Unfettered Hai. Not their pointless war - she hated that - but their drive to progress and their instinct for change. Our family tried to discourage her ideas, and we thought we succeeded, but... maybe...`
+			`	"A trauma surgeon for the Unfettered Hai? I wonder if she planned this years in advance?"`
+				goto questions
+			label accept
+			`	"If Anoo went anywhere, it would be Darkcloak. The largest Unfettered hospital is there. Many soldiers come back from raids injured on Darkcloak. A trauma surgeon would have a lot of work to do."`
+				accept
+			
+mission "Acorn Delights: Part 5"
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	landing
+	source Darkcloak
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 4: done"
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+	on offer
+		"salary: Acorn Delights Dividends" = 500
+		payment 30000
+		log `Helped the famous Hai chef Eeeya find her granddaughter, a trauma surgeon named Anoo. Both joined the Unfettered hoping they can, "set them on a more productive path." Received shares of Acorn Delights stock which pay 500 credits a day in dividends.`
+		conversation
+			`You're ushered quickly from the landing platform to a nearby hospital. Unfettered Hai nearby are muttering in their language, but you keep hearing the Hai word for, "cookies," as they eye Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. It seems the Unfettered value function and progress, rather than aesthetics and comfort like their southern brethren.`
```

Comment:
` but you keep hearing the Hai word for, "cookies," as they eye Eeeya.`
not sure if this punctuation is correct.

Reply:
Fixed.